rules/eslint.js

```diff
@@ -74,6 +74,7 @@ export default {
     // Conflicts with unicorn/expiring-todo-comments.
     'no-warning-comments': 'off',
     'one-var': ['error', 'never'],
+    'prefer-exponentiation-operator': 'error',
```

Comment:
this rule got deprecated in `unicorn` plugin and is now part of ESLint rules, so moved it and its fixtures